charts/templates/configmap.yaml

```diff
@@ -10,3 +10,130 @@ data:
   MARKLOGIC_FQDN_SUFFIX: {{ include "marklogic.headlessURL" . }}
   MARKLOGIC_INIT: "true"
   MARKLOGIC_JOIN_CLUSTER: "true"
+---
```

Comment:
missing an if statement. When logCollection.enabled is false, we don't want to deployment the configmap for fluent-bit.